sklearn/linear_model/tests/test_sgd.py

```diff
@@ -1542,7 +1542,7 @@ def test_gradient_squared_epsilon_insensitive():
 def test_multi_thread_multi_class_and_early_stopping():
     # This is a non-regression test for a bad interaction between
     # early stopping internal attribute and thread-based parallelism.
-    clf = SGDClassifier(alpha=1e-3, tol=1e-3, max_iter=1000,
```

Comment:
This needs to be reverted because:

https://github.com/scikit-learn/scikit-learn/blob/1a14920464acd9c16fcc0187f89cbbd738068ebc/sklearn/linear_model/tests/test_sgd.py#L75-L77

Changes the `max_iter=5` for all tests in this file.